src/DependencyInjection/Configuration.php

```diff
@@ -102,6 +102,13 @@ public function getConfigTreeBuilder()
                                 ->thenInvalid('Configuration value of "global_search.empty_boxes" must be one of show, fade or hide.')
                             ->end()
                         ->end()
+                        ->booleanNode('case_sensitive')
+                            ->defaultTrue()
+                            ->info('Whether the global search should behave case sensitive or not.
+                                Using case-insensitivity might lead to performance issues.
+                                See https://use-the-index-luke.com/sql/where-clause/functions/case-insensitive-search
+                                for more information.')
```

Comment:
Should you use nowdoc here? Otherwise the `debug:config` will look a bit weird, won't it?

Reply:
TBH I didn't checked it ;-)

Reply:
![pitchforks](https://user-images.githubusercontent.com/657779/50564774-2b6e1b80-0d28-11e9-8120-da93aa6677e4.jpg)

Reply:
done